### ISSUE TITLE
Translation for Danish (CLI)

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#223]
There is one message that needs update. Can you provide it in a comment here or a pull request?

```
msgid ""
"Source IP address used to send queries. Setting an IP address not correctly "
"configured on a local network interface causes cryptic error messages."
msgstr ""
```

**Comment by matsduf** [comment]
@haarbo, can you provide the translation to this?

**Comment by matsduf** [comment]
Resolved by #236
